README.md

```diff
@@ -1,3 +1,11 @@
+## Deprecation note!
+
+**Please note this repository is deprecated - this repository is no longer actively maintained**. 
+
+However, this library is part of Arm's [Mbed Cloud](https://cloud.mbed.com/docs/current) -offering and as such the code is available via [Mbed Cloud Client](https://github.com/ARMmbed/mbed-cloud-client-example) under
```

Comment:
instead of mbed-cloud-client-example this should link to mbed-cloud-client repository.

Reply:
Why is there - before "offering"?
Also, you can shorten it to "offering, and the code is available in [give just one link, directly to the correct folder]